front/app/components/UI/QuillEditor/altTextToImagesModule.tsx

```diff
@@ -15,17 +15,14 @@ export class ImageBlot extends BlockEmbed {
   // Instead of using the default img tag, we are using a div where we can add an input field for the alt text
   static tagName = ['div'];
 
-  static create(value: { alt: string; src: string } | string) {
+  static create(value: string) {
```

Comment:
We've simplified the create method a bit and deleted the value method to make sure copy/paste works as expected.